src/applications/hca/config/form.js

```diff
@@ -432,6 +440,66 @@ const formConfig = {
                   },
                 },
               }),
+              'view:doesPermanentAddressMatchMailing': {
+                type: 'boolean',
+              },
+            },
+          },
+        },
+        mailingAddress: {
+          path: 'veteran-information/mailing-address',
+          title: 'Mailing address',
+          initialData: {},
+          depends: formData =>
```

Comment:
Mailing Address page for second address that is being conditionally rendered on line 453.

For this page to show the feature flip must be true on line 454 as well as the yes/no must be false for the addresses being the same.